packages/ui/src/theme/roots/palette/entity/default-palette.ts

```diff
@@ -51,38 +51,44 @@ export const lightPalette = {
   [Color.TextSecondaryInvert]: 'rgba(213,216,220,1)',
   [Color.TextPlaceholder]: 'rgba(181,188,190,1)',
 
-  [Color.AccentPrimary]: 'rgba(104,121,235,1)',
-  [Color.AccentPrimaryHover]: 'rgba(96,111,209,1)',
-  [Color.AccentPrimaryActive]: 'rgba(73,88,188,1)',
-  [Color.AccentPrimaryDisabled]: '#E9EEF2',
+  [Color.AccentPrimary]: '#6879EB',
+  [Color.AccentPrimaryHover]: '#606FD1',
+  [Color.AccentPrimaryActive]: '#4958BC',
+  [Color.AccentPrimaryDisabled]: 'rgba(104, 121, 235, 0.12)',
 
   [Color.BackgroundPrimary]: 'rgba(248,250,251,1)',
   [Color.BackgroundPrimaryHover]: 'rgba(239,243,246,1)',
   [Color.BackgroundPrimaryActive]: 'rgba(93,109,126,1)',
   [Color.BackgroundPrimary_0_5_Opacity]: 'rgba(255,255,255,0.500)',
   [Color.BackgroundSecondary]: 'rgba(255,255,255,1)',
 
-  [Color.IconsPrimary]: 'rgba(93,109,126,1)',
+  [Color.IconsPrimary]: '#6879EB',
   [Color.IconsPrimaryOpacity_0_6]: 'rgba(93,109,126,0.600)',
   [Color.IconsSecondary]: '#959DA8',
   [Color.IconsPlaceholder]: '#DEE6ED',
+  [Color.IconButtonBasicHoverColor]: ' #4f6276',
+  [Color.IconButtonBasicHoverBackground]: ' #f9fbfc',
+  [Color.IconButtonBasicActiveColor]: ' #37414b',
```

Comment:
Actually we don't need this new colors, we should use the same color used in figma. If you click an item in figma you will see the name of the color used and it's value. If you find the same name in UI kit but different value from figma you should update it in UI kit to the latest value in figma

So please check all colors that you added and check again with figma